cmd/vclusterctl/cmd/completion.go

```diff
@@ -0,0 +1,92 @@
+package cmd
+
+import (
+	"errors"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+const compDesc = `
+#######################################################
+################### vcluster complete #################
+#######################################################
+Generates completion scripts for various shells
+
+Example:
+vcluster completion bash
+vcluster completion zsh 
+#######################################################
+`
+
+const bashCompDesc = `
+#######################################################
+################### vcluster complete bash ############
+#######################################################
+Generate the autocompletion script for bash
+
+Example:
+- Linux:
+vcluster completion bash > /etc/bash_completion.d/vcluster
+- MacOS:
+vcluster completion bash > \ 
```

Comment:
Could you remove the space after '\\' as this will lead to the command executing directly instead of breaking the line